test/TestServerProjects/xml-service/Generated/Models/Blobs.Serialization.cs

```diff
@@ -24,9 +24,9 @@ internal static Blobs DeserializeBlobs(XElement element)
             }
             blobPrefix = array;
             var array0 = new List<Blob>();
-            foreach (var e0 in element.Elements("Blob"))
+            foreach (var e in element.Elements("Blob"))
```

Comment:
This got better!

Reply:
Is there a reason it is `array0` here? Do we need to apply the same fix for arrays?

Reply:
It correct, two arrays are defined in the same scope.

It's mostly the result of the way XML deserialization is done, we can improve names by prefixing them with serialized name so we get something like `blobPrefixArray` but that's a different change :)